### PR TITLE
Create new Groups Directory block for the Block Editor

### PR DESCRIPTION
**Pull request by reygcalantaol**
@polevaultweb

### CODE REVIEW

**Comment by reygcalantaol** [comment]
@polevaultweb 
This PR is related to 
https://github.com/WPUserManager/wpum-groups/issues/4

File: includes/classes/Loader.php [hunk 1]
```diff
@@ -51,6 +52,7 @@ public function register() {
 		( new RecentlyRegisteredUsers() )->register();
 		( new RegistrationForm() )->register();
 		( new UserDirectory() )->register();
+		( new GroupDirectory() )->register();
```
Comment (polevaultweb) [inline includes/classes/Loader.php:55]: We should only load this block if the Groups addon is active

File: includes/classes/Blocks/GroupDirectory.php [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+		];
+	}
+
+	/**
+	 * Retrieve the list of attributes.
+	 *
+	 * @return array
+	 */
+	protected function get_attributes() {
+		return [];
```
Comment (polevaultweb) [inline includes/classes/Blocks/GroupDirectory.php:43]: We should be able to control the shortcode settings via the block panel
```
'per_page' => 10,
'has_search_form' => 'true',
				'show_public' => 'true',
				'show_private' => 'true',
```

**Review by polevaultweb — CHANGES_REQUESTED** [review]
@reygcalantaol tested and works great, some comments added

File: includes/classes/Loader.php (hunk 1 above)
Comment (polevaultweb) [reply includes/classes/Loader.php:55]: Check for a function or class exists is probably easiest

**Comment by polevaultweb** [comment]
@reygcalantaol did you see my review?

**Comment by reygcalantaol** [comment]
@polevaultweb 
I pushed new commits for the updates.

**Pull request merged**